redaxo/src/core/pages/system.settings.php

```diff
@@ -206,10 +206,16 @@
             <th>PHP</th>
             <td>' . PHP_VERSION . ' <a class="rex-link-expanded" href="' . rex_url::backendPage('system/phpinfo') . '" title="phpinfo" onclick="newWindow(\'phpinfo\', this.href, 1000,800,\',status=yes,resizable=yes\');return false;"><i class="rex-icon rex-icon-phpinfo"></i></a></td>
         </tr>
+        <tr>
+            <th class="rex-table-width-3">'.rex_i18n::msg('path').'</th>
```

Comment:
```suggestion
            <th>'.rex_i18n::msg('path').'</th>
```

Die class kann raus, da diese bereits im ersten `tr` gesetzt wurde.

Reply:
Danke